### DISCUSSION TITLE
Does Enforce 100% Fill Volume affect gap fill?

### DISCUSSION THREAD

**Discussion by blex-max** [supermerill/SuperSlicer#2069] [Q&A] [answered]
Quick and easy question, as per the title!

**Comment by supermerill** [comment, marked as answer]
It only applies to solid (in)fill.
So yes but only gapfill from `(filled)` infill pattern.
Perimeters & perimeter gapfill should already be exactly at 100% (unless you reduced the overlap)

Note that now, if you decrease the filament max overlap then the entire print is sightly under-extruded, but it's for the greater good.
